CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Features
+
+- Add an option to disable native (iOS and Android) profiling for the `HermesProfiling` integration ([#4094](https://github.com/getsentry/sentry-react-native/pull/4094))
+
```

Comment:
This is a suggestion and not a request for a change. Being new to the project I wonder if we could also add a usage example in the changelog.

Reply:
certainly a great suggestion, I'll add it!